config/local/discoverable-client.yml

```diff
@@ -17,12 +10,27 @@ server:
         keyAlias: localhost
         keyPassword: password
         keyStore: keystore/localhost/localhost.keystore.p12
-        # keyStore: keystore/selfsigned/localhost.keystore.p12        
-        # keyStore: keystore/selfsigned/localhost-untrusted.keystore.p12
         keyStorePassword: password
-        keyStoreType: PKCS12
         trustStore: keystore/localhost/localhost.truststore.p12
-        # trustStore: keystore/selfsigned/localhost.truststore.p12
-        # trustStore: keystore/selfsigned/localhost-untrusted.truststore.p12
         trustStorePassword: password
-        trustStoreType: PKCS12
+
+apiml:
+    enabled: true
+
+    service:
```

Comment:
This file should be just override for local execution and should not contain redundant or irrelevant nodes